Loop/Managers/LoopDataManager.swift

```diff
@@ -1298,6 +1367,15 @@ protocol LoopDataManagerDelegate: class {
     ///   - units: The recommended bolus in U
     /// - Returns: a supported bolus volume in U. The volume returned should not be larger than the passed in rate.
     func loopDataManager(_ manager: LoopDataManager, roundBolusVolume units: Double) -> Double
+
+    /// The pump manager status, if one exists.
+    var pumpManagerStatus: PumpManagerStatus? { get }
```

Comment:
In Loop's current architecture, DeviceDataManager mediates interaction with the pump, which allows LoopDataManager to limit its responsibility to just knowledge of delivery via DoseStore.  `basalDeliveryState` is one other piece of information about the pump that DDM passes to LDM, and this creates a possibly conflicting redundancy with that as well. I would recommend that this code is change so that DDM has responsibility for providing pumpmanager status for a status store. I think this means that DDM would manage the status store, which is more in line with the current architecture, as DDM "owns" LDM.